server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableUtils.java

```diff
@@ -650,4 +649,17 @@ public static MajorCompactionReason from(CompactionKind ck) {
         throw new IllegalArgumentException("Unknown kind " + ck);
     }
   }
+
+  public static TabletFile computeCompactionFileDest(TabletFile tmpFile) {
+    String newFilePath = tmpFile.getMetaInsert();
+    int idx = newFilePath.indexOf("_tmp");
+    if (idx > 0) {
+      newFilePath = newFilePath.substring(0, idx);
+    } else {
+      throw new RuntimeException(
+          "Expected compaction tmp file " + tmpFile.getMetaInsert() + " to have suffix '_tmp'");
```

Comment:
Is there a more specific RTE that can be used? IllegalArgumentException might be appropriate, based on this error message.

Reply:
changed in f75c2a8e4804685effc217c680e8cb03b852768a